.gitignore

```diff
@@ -4,3 +4,4 @@ kore
 *.module
 *.DSYM
 cert
+.idea
```

Comment:
Ok, I think that it's better to not using CLion for Kore contributions. I will fix those style issues.